api-generation/src/main/scala/ai/h2o/sparkling/api/generation/common/IgnoredParameters.scala

```diff
@@ -33,12 +33,13 @@ object IgnoredParameters {
   def all(algorithm: String): Seq[String] = implementedInParent ++ deprecated(algorithm) ++ {
     algorithm match {
       case "H2OGAM" =>
-        Seq("plug_values", "interaction_pairs") // According to MK the parameter doesn't make much sense for GAM
+        Seq(
+          "plug_values", // According to MK the parameter doesn't make much sense for GAM
+          "interaction_pairs") // Interaction pairs are not currently supported on MOJO
       case "H2ODeepLearning" => Seq("pretrained_autoencoder", "checkpoint")
       case "H2OGBM" => Seq("checkpoint")
       case "H2ODRF" => Seq("checkpoint")
       case "H2OXGBoost" => Seq("checkpoint")
-      case "H2OGLM" => Seq("interaction_pairs")
```

Comment:
Has to be exposed even if they are not currently supported by H2O-3 since The parameter has already been exposed in previous SW versions.